app/controllers/work_packages/activities_tab_controller.rb

```diff
@@ -255,7 +255,9 @@ def handle_other_filters_on_create(call)
   end
 
   def perform_update_streams_from_last_update_timestamp
-    if params[:last_update_timestamp].present? && (last_updated_at = Time.zone.parse(params[:last_update_timestamp]))
+    last_update_timestamp = params[:last_update_timestamp] || params.dig(:journal, :last_update_timestamp)
+
+    if last_update_timestamp.present? && (last_updated_at = Time.zone.parse(last_update_timestamp))
```

Comment:
I don't really like attributions on if statements. I wonder if this would change behaviour.

```ruby
if last_update_timestamp.present?
  last_updated_at = Time.zone.parse(last_update_timestamp)
  ...
end
```

In any case, it was already like this, so treat this comment as an optional nitpick :sweat_smile: